index.html

```diff
@@ -6,13 +6,23 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-    <div id="container">
-        <header id="header">Header A</header>
-        <main id="main">Main B</main>
-        <aside id="big-aside">big aside C</aside>
-        <aside id="small-aside">Small aside D</aside>
-        <p id="paragraph">Paragraph E</p>
-        <footer id="footer">Footer F</footer>
-    </div>
+<div id="container">
```

Comment:
in de uitwerkingen wordt er nog gebruik gemaakt van een class "items" zodat je ze generiek kan stylen.

in de CSS kun je deze "items" een flexbox mee geven zodat je A t/m F centraal kan alignen.